docs/src/block-editor/block-classes.md

```diff
@@ -0,0 +1,27 @@
+---
+pageClass: twill-doc
+---
+
+# Block classes
+
+If you need more control over blocks, their validation or data for rendering you can use a block class.
+
+To do this create a file named after your block. (ex. for `hero_header_with_menu.blade.php` your class will be 
```

Comment:
This is really nitpicking but I feel like that example might not be the most relevant as blocks are not necessarily the right tool to build a hero header with a menu. Let's use something like `images_grid` and `ImagesGrid` instead?